src/ripple/ledger/impl/ApplyStateTable.cpp

```diff
@@ -596,14 +596,12 @@ ApplyStateTable::threadTx (ReadView const& base,
 {
     auto const sle = getForMod(base,
         keylet::account(to).key, mods, j);
-    assert(sle);
     if (! sle)
     {
-        // VFALCO We need to think about throwing
-        //        an exception or calling LogicError
-        JLOG(j.fatal()) <<
-            "Threading to non-existent account: " <<
-                toBase58(to);
+        // The Destination of an Escrow or PayChannel may have been deleted.
```

Comment:
Dst of escrow cannot be deleted

Reply:
Follow-up on my own comment. I'm wrong about the above. I forgot about pre-fix escrows. So no change needed.